manager-bundle/tests/Command/ContaoSetupCommandTest.php

```diff
@@ -63,7 +63,7 @@ public function testExecutesCommands(array $options, array $flags, array $phpFla
 
         $createProcessHandler = $this->getCreateProcessHandler($processes, $commandArguments, $invocationCount);
 
-        $command = new ContaoSetupCommand('project/dir', 'project/dir/public', $createProcessHandler);
+        $command = new ContaoSetupCommand('project/dir', 'project/dir/public', $createProcessHandler, '');
```

Comment:
Why `''` instead of `null`?

Reply:
This makes the test ignore the setting, while `null` would make it read from the current configuration (fallback, see constructor).

Reply:
Can't you set a real value such as `-1` or `1G` instead?

Reply:
This test basically tests the `empty` statement (see below) while `testSetsMemoryLimitForSubProcesses()` sets + tests a value. We could use 1G there instead of `<memory-limit>` if you prefer?

```php
if (!empty($this->memoryLimit)) {
    $php[] = "-dmemory_limit=$this->memoryLimit";
}
```

Reply:
My point is that `''` is not a valid (or at least reasonable) value. It is just a workaround for the unit test, isn't it?

Reply:
I think you can set anything: `php -dmemory_limit='' -r "echo ini_get('memory_limit') . PHP_EOL;"` outputs an empty string but I don't know what this would do effectively.

I think the unit test is correct as long as the `empty()` check is in there. But I am tempted to remove it altogether. Then we can drop the other test and just set it to`1G`.  Wdyt?

Reply:
Like it. 👍 

Are you sure that `1G` is enough? The Contao Manager will try to increase the memory limit to 1.5G:

https://github.com/contao/contao-manager/blob/981e239354ef2e230b0e1e045add7acbb59a8ef4/api/console#L52-L55

Reply:
The `1G` setting is only for the test that only creates mocks, so this is fine. 
Hopefully, we're talking about the same thing. :see_no_evil: 

Here are the changes, now quite minimal: a623be39930c351cad0ecfd92e34fea45463f64d